va/tlsalpn.go

```diff
@@ -48,6 +48,10 @@ func certNames(cert *x509.Certificate) []string {
 	}
 	names = append(names, cert.DNSNames...)
 	names = core.UniqueLowerNames(names)
+	// TODO: This for loop can be deleted after new builds of boulder use
```

Comment:
Add an issue number here, e.g. `TODO(#123)`

Reply:
Thanks, done in both places.